pages/packages.md

```diff
@@ -49,14 +49,14 @@ Some advanced package creation tools include:
 
 Currently, Buildkite Packages supports the following package ecosystems:
 
-- Alpine
-- Docker container images
-- Debian (deb for Debian and Ubuntu)
-- Ruby (RubyGems gem)
+- Alpine (apk)
+- Container (Docker) images
+- Debian (and Ubuntu)
 - Java (Maven and Gradle)
-- JavaScript (Node.js npm)
+- JavaScript (NPM)
 - Python (PyPI)
-- RPM (Fedora and RHEL)
+- RedHat (RPM)
```

Comment:
It could be worth mentioning Fedora here since that feels like the more well-known name. e.g. maybe `RedHat and Fedora (RPM)`